components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -2701,6 +2738,11 @@ void RewardsServiceImpl::SetShortRetries(bool short_retries) {
   bat_ledger_service_->SetShortRetries(short_retries);
 }
 
+void RewardsServiceImpl::SetCurrentCountry(
+    const std::string& current_country_) {
```

Comment:
underscore at the end of variables should only be used for class member variables